package.json

```
@@ -60,7 +60,7 @@
     "@typescript-eslint/eslint-plugin": "^5.43.0",
     "eslint": "^8.0.1",
     "eslint-plugin-import": "^2.25.2",
-    "eslint-plugin-n": "^15.0.0",
+    "eslint-plugin-n": "^16.0.0",
```

```suggestion
    "eslint-plugin-n": "^15.0.0 || ^16.0.0 ",
```

https://github.com/standard/eslint-config-standard/blob/master/package.json